### PR TITLE
Updating to v1.0.0 with  node mapnik v3.7.2

### PR DESCRIPTION
**Pull request by thesocialdev**
Updating libraries to match node mapnik v3.72 on kartotherian and its dependencies

### CODE REVIEW

**Comment by thesocialdev** [comment]
@mdholloway this PR is blocked until all the libraries are published, but if you are going to test you should use this code.

**Comment by mdholloway** [comment]
To get the build passing, looks like we'll need to have the Docker build download the backport of mapnik 3.0.20 from apt.wikimedia.org.  I think you'll need to put back and update these two lines: https://github.com/kartotherian/kartotherian/blob/ddc9f61fad67f21584e15b381cefd521b27abb96/Dockerfile#L3-L4 
which were recently removed in https://github.com/kartotherian/kartotherian/commit/e624b704f12cf69312b0dee0094017e4d364e0f0#diff-3254677a7917c6c01f55212f86c57fbf.

**Comment by mdholloway** [comment]
Please disregard the last comment, I don't seem to have been looking at the most recent version of this PR. ;)

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,8 @@
 FROM debian:stretch
-RUN apt-get update && apt-get install -y nodejs nodejs-legacy git wget python libcairo2-dev libjpeg62-turbo-dev libpango1.0-dev libgif-dev build-essential g++ libmapnik-dev && rm -rf /var/lib/apt/lists/*
+RUN apt-get update && apt-get install -y nodejs nodejs-legacy git wget python apt-transport-https libboost-filesystem-dev libboost-program-options-dev libboost-regex-dev libboost-system-dev libboost-thread-dev libxml2 libxml2-dev libfreetype6 libfreetype6-dev libjpeg-dev libpng-dev libtiff5-dev libcairo2 libcairo2-dev ttf-unifont ttf-dejavu ttf-dejavu-core ttf-dejavu-extra build-essential libgdal-dev libharfbuzz-bin libharfbuzz-dev libmapnik-dev && rm -rf /var/lib/apt/lists/*
```
Comment (mdholloway) [inline Dockerfile]: It is not necessary to explicitly install most of these; they'll be pulled in as needed by libmapnik3.0.

**Review by mdholloway — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -37,28 +37,29 @@
...
+    "@kartotherian/tilelive-tmsource": "~1.0.0",
+    "@kartotherian/tilelive-vector": "^4.0.0",
     "@mapbox/tilejson": "^1.0.3",
+    "@mapbox/tilelive": "~5.12.2",
+    "@mapbox/tilelive-bridge": "~3.1.0",
     "bluebird": "^3.5.0",
     "body-parser": "^1.17.1",
     "bunyan": "^1.8.9",
     "cassandra-uuid": "^0.0.2",
+    "colors": "^1.3.1",
```
Comment (mdholloway) [inline package.json]: This looks like a new dependency.  Where is it used?

**Review by mdholloway — CHANGES_REQUESTED** [review]

**Review by mdholloway — APPROVED** [review]

**Pull request merged**